content/company/about-us.mdx

```diff
@@ -4,17 +4,12 @@ seo:
 sidebar: |
   <MicrosoftPanel />
-sidebarTestimonial: content/testimonials/System77.mdx
+sidebarTestimonial: >-
```

Comment:
change the schema to be an object instead of a string

Reply:
Not done - as discussed yesterday, to handle this as a dropdown field, I'm passing a name of the testimonial as a value of dropdown field.